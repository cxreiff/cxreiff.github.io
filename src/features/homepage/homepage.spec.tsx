import React from 'react'
import { Provider } from 'react-redux'
import configureMockStore from 'redux-mock-store'
import { shallow } from 'enzyme'

import Homepage from './homepage.component'

describe('homepage', () => {

    const mockStore = configureMockStore()

    afterEach(() => {
        jest.restoreAllMocks()
    })

    it('should render', () => {
        const store = mockStore({
          counter: 0,
        })
        const component = shallow(
            //@ts-ignore
            <Provider store={store}>
                <Homepage />
            </Provider>
        )
        expect(component).toMatchSnapshot()
    })
})
