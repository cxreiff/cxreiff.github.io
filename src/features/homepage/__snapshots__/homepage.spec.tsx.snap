// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`homepage should render 1`] = `ShallowWrapper {}`;
